src/services/auth-service.ts

```diff
@@ -18,13 +18,20 @@ export class AuthService {
     return !this.isUserSignedIn();
   }
 
-  async signInWithPassword(data: { email: string; password: string }, onSuccess: () => void, onError: (error: AuthError) => void = (error) => console.warn('Encountered an error when signing in.', error.cause)) {
+  async signInWithPassword(data: { email: string; password: string }, onSuccess: () => void, onError: (error: AuthError) => void = (error) => {
+    console.warn('Encountered an error when signing in.', error.cause);
+    alert(error.message);
+  }) {
     const { error } = await this.supabase.auth.signInWithPassword({
       email: data.email,
       password: data.password,
     });
 
-    return error ? onError(error) : onSuccess();
```

Comment:
Nitpick: I think by removing `return` we are now not actually returning anything from this function. Given they currently return `void` that's probably fine, but it might be best to keep the `return` in there to be explicit.

Not sure what best practice is though... @killerdaisies any thoughts/opinions?